### PR TITLE
Frontend route fix

### PR DESCRIPTION
**Pull request by yousiefc**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

- Completed a submission in the application

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/efiling-demo/README.md [hunk 1]
```diff
@@ -4,9 +4,15 @@ Welcome to the eFiling demo!
 
 ## Running the Frontend
 
+In the project directory (`efiling-demo`):
```
Comment (alexjoybc) [inline src/frontend/efiling-demo/README.md:7]:
```suggestion
In the project directory `efiling-demo`:
```

File: src/frontend/efiling-demo/README.md [hunk 2]
```diff
@@ -4,9 +4,15 @@ Welcome to the eFiling demo!
...
+
 Set the required environment variables locally. Create a `.env.development` file and populate the environment variables as shown in the `.env.example` file.
 
-In the project directory (`efiling-demo`), you can run:
+In the root directory (`jag-file-submission`), run:
```
Comment (alexjoybc) [inline src/frontend/efiling-demo/README.md:11]:
```suggestion
In the root directory `jag-file-submission`, run:
```

File: src/frontend/efiling-demo/README.md [hunk 3]
```diff
@@ -4,9 +4,15 @@ Welcome to the eFiling demo!
...
+
+### `docker-compose up efiling-api efiling-frontend keycloak keycloak-config redis`
```
Comment (alexjoybc) [inline src/frontend/efiling-demo/README.md:13]:
```suggestion
```bash
docker-compose up efiling-api efiling-frontend keycloak keycloak-config redis
```
```

cmd should be wrapped in ```

File: src/frontend/efiling-demo/README.md [hunk 4]
```diff
@@ -4,9 +4,15 @@ Welcome to the eFiling demo!
...
+
+In the project directory (`efiling-demo`) open a new terminal and run:
 
 ### `yarn install`
```
Comment (alexjoybc) [inline src/frontend/efiling-demo/README.md:17]: Could you also update those ones, I think the recommendation is to wrap any cmd in ``` I don't know why the react boilerplate puts it like that...

File: src/frontend/efiling-demo/README.md [hunk 5]
```diff
@@ -15,7 +21,7 @@ Installs all the required dependencies to get the application up and running.
 ### `yarn start`
 
 Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Open [http://localhost:3001](http://localhost:3001) to view it in the browser.
```
Comment (alexjoybc) [inline src/frontend/efiling-demo/README.md:24]: nice

File: src/frontend/efiling-demo/package.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "efiling-demo",
   "version": "0.6.0",
-  "homepage": "/efiling-demo",
+  "homepage": ".",
```
Comment (alexjoybc) [inline src/frontend/efiling-demo/package.json:4]: question: is the `.` required ? I think it is but cannot remember why...

**Review by alexjoybc — CHANGES_REQUESTED** [review]
Nice work @yousiefc did you check that the app still runs on docker, we don't have our integration test fully fixed, so just want to make sure it still runs on `docker-compose up` ;-)
I did put a couple of suggestion for the readme

File: src/frontend/efiling-demo/package.json (hunk 6 above)
Comment (yousiefc) [reply src/frontend/efiling-demo/package.json:4]: Yes this was the only way it worked

**Comment by yousiefc** [comment]
I've made all the changes it should be good to go

File: src/frontend/efiling-demo/package.json (hunk 6 above)
Comment (alexjoybc) [reply src/frontend/efiling-demo/package.json:4]: Sounds good ;-)

**Review by alexjoybc — APPROVED** [review]
Awesome work @yousiefc thanks for the contribution!!

**Pull request merged**